packages/core/src/enums/ViewportType.ts

```diff
@@ -15,6 +15,7 @@ enum ViewportType {
   ORTHOGRAPHIC = 'orthographic',
   /** Perspective Viewport: Not Implemented yet */
   PERSPECTIVE = 'perspective',
+  VOLUME_3D = 'volume3d',
```

Comment:
I don't think VOLUME_3D is a good pick, but cannot think of a better one either. I think by default we have camera parallel projection ON

@swederik Do you think it should be perspective 3D? or orthogonal 3D makes sense, also any idea for a good name?

Reply:
Yes, I couldn't think of a better name. I'll wait for your input on this.